packages/block-library/src/image/image.js

```diff
@@ -94,6 +102,216 @@ const ImageWrapper = ( { href, children } ) => {
+								) }
+							</ToolbarButton>
+						) }
+						renderContent={ () => (
+							<TextareaControl
+								className="wp-block-image__toolbar_content_textarea"
+								label={ __( 'Alternative text' ) }
+								value={ attributes.alt || '' }
+								onChange={ ( value ) =>
+									setAttributes( { alt: value } )
+								}
+								disabled={ lockAltControls }
+								help={
+									lockAltControls ? (
+										<>{ lockAltControlsMessage }</>
+									) : (
+										<>
+											<ExternalLink
+												href={
+													// translators: Localized tutorial, if one exists. W3C Web Accessibility Initiative link has list of existing translations.
+													__(
+														'https://www.w3.org/WAI/tutorials/images/decision-tree/'
+													)
+												}
+											>
+												{ __(
+													'Describe the purpose of the image.'
+												) }
+											</ExternalLink>
+											<br />
+											{ __(
+												'Leave empty if decorative.'
+											) }
+										</>
+									)
+								}
+								__nextHasNoMarginBottom
+							/>
+						) }
+					/>
+				) }
+				{ activeControls.includes( 'title' ) && (
+					<Dropdown
+						defaultOpen={
+							! previousActiveControls?.includes( 'title' )
+						}
+						popoverProps={ WRITEMODE_CONTROLS_POPOVER_PROPS }
+						renderToggle={ ( { isOpen, onToggle } ) => (
+							<ToolbarButton
+								onClick={ onToggle }
+								aria-haspopup="true"
+								aria-expanded={ isOpen }
+								onKeyDown={ ( event ) => {
+									if ( ! isOpen && event.keyCode === DOWN ) {
+										event.preventDefault();
+										onToggle();
+									}
+								} }
+							>
+								{ __( 'Title' ) }
+							</ToolbarButton>
+						) }
+						renderContent={ () => (
+							<TextControl
+								__next40pxDefaultSize
+								className="wp-block-image__toolbar_content_textarea"
+								__nextHasNoMarginBottom
+								label={ __( 'Title attribute' ) }
+								value={ attributes.title || '' }
+								onChange={ ( value ) =>
+									setAttributes( { title: value } )
+								}
+								disabled={ lockTitleControls }
+								help={
+									lockTitleControls ? (
+										<>{ lockTitleControlsMessage }</>
+									) : (
+										<>
+											{ __(
+												'Describe the role of this image on the page.'
+											) }
+											<ExternalLink href="https://www.w3.org/TR/html52/dom.html#the-title-attribute">
+												{ __(
+													'(Note: many devices and browsers do not display this text.)'
+												) }
+											</ExternalLink>
+										</>
+									)
+								}
+							/>
+						) }
+					/>
+				) }
+			</BlockControls>
+		);
+	}
+
+	return (
+		// Add some extra controls for content attributes when content only mode is active.
+		// With content only mode active, the inspector is hidden, so users need another way
+		// to edit these attributes.
+		<>
+			{ controls }
+			<BlockControls group="inline">
```

Comment:
If we have both the `more` menu and the active toolbar buttons in the same BlockControls group, they can be rendered after the menu. I'm not sure if we have so fine control over the order of fills in this slot, but I think not..